### PR TITLE
Node 18+ updates

### PR DESCRIPTION
**Pull request by fajar-apri-alaska**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Resolves:** #44, #45, #46, #47

## Summary:

...

## Type of change:

Please delete options that are not relevant.

- [x] Revision of an existing capability

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

**Review by blackfalcon — COMMENTED** [review]
Please delete `./index.html` from the root of the project.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    attributes:
+      value: >
+        [![See it on
+        NPM!](https://img.shields.io/npm/v/@aurodesignsystem/auro-flightline?style=for-the-badge&color=orange)](https://www.npmjs.com/package/@aurodesignsystem/auro-flightline)
+  - type: textarea
+    id: details
+    attributes:
+      label: Please describe the bug
+      description: List out the steps to reproduce the behavior
+      placeholder: 1. Go to '...' 2. Click on '....' 3. Scroll down to '....' 4. See error
```
Comment (blackfalcon) [inline .github/ISSUE_TEMPLATE/bug_report.yml:27]: Please replace with the following 

```
      placeholder: |
        1. Go to '...'
        2. Click on '....'
        3. Scroll down to '....'
        4. See error
```

File: .github/settings.yml [hunk 2]
```diff
@@ -65,10 +66,13 @@ branches:
 # Labels: define labels for Issues and Pull Requests
 labels:
```
Comment (blackfalcon) [inline .github/settings.yml:67]: Missing content

```
labels:
  - name: auro-flightline
    color: 'fa23e4'
    description:
    process: Key filter for this repo
```

**Review by blackfalcon — CHANGES_REQUESTED** [review]

File: demo/index.html [hunk 3]
```diff
@@ -28,6 +28,10 @@
         });
     </script>
     <script type="module" src="../src/auro-flightline.js"></script>
+    <script type="module">
+      import { registerComponent } from "../index.js"
+      registerComponent('custom-flightline');
+    </script>
```
Comment (blackfalcon) [inline demo/index.html:34]: this is not needed in this file

**Review by blackfalcon — CHANGES_REQUESTED** [review]

**Review by blackfalcon — CHANGES_REQUESTED** [review]

**Review by blackfalcon — COMMENTED** [review]
This commit is not prefixed correctly. For one, I am confused as to why adding line breaks to the code is needed as a single commit? 

Also, this is an update to the docs, so the correct prefix would be `docs`, not fix.

File: .github/workflows/testPublish.yml [hunk 4]
```diff
@@ -24,10 +24,10 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
       - run: npm ci
-      - run: npm run sassBuild:component
+      - run: npm run build:sass:component
```
Comment (blackfalcon) [inline .github/workflows/testPublish.yml:27]: Please see how the testPublish document is written in the generator. 

https://github.com/AlaskaAirlines/WC-Generator/blob/master/template/.github/workflows/testPublish.yml

**Review by blackfalcon — CHANGES_REQUESTED** [review]

File: package.json [hunk 5]
```diff
@@ -33,57 +33,58 @@
...
-    "rollup": "^2.59.0",
-    "rollup-plugin-serve": "^1.1.0",
-    "sass": "^1.57.1",
-    "semantic-release": "^19.0.5",
-    "sinon": "^15.0.1",
-    "stylelint": "^14.16.1",
-    "stylelint-config-standard": "^29.0.0",
+    "rollup": "^3.21.5",
+    "rollup-plugin-serve": "^2.0.2",
+    "sass": "^1.62.1",
```
Comment (blackfalcon) [inline package.json:80]: Please add support for Sass override 

https://github.com/AlaskaAirlines/WC-Generator/blob/master/template/package.json#L75-L79

**Review by blackfalcon — CHANGES_REQUESTED** [review]

File: package.json [hunk 6]
```diff
@@ -7,7 +7,7 @@
     "  3. Go to http://localhost:3001",
     "================================================================================"
   ],
-  "name": "@aurodesignsystem/auro-flightline",
+  "name": "@alaskaairux/auro-flightline",
```
Comment (blackfalcon) [inline package.json]: You were correct to add update to `@aurodesignsystem` in the 1st place. Please add this back and remember to use `BREAKING CHANGE` in the body of the commit.

**Review by blackfalcon — CHANGES_REQUESTED** [review]

File: package.json (hunk 6 above)
Comment (blackfalcon) [reply package.json]: Looks like this was updated in a later commit... disregard.

File: src/auro-flightline.js [hunk 7]
```diff
@@ -4,8 +4,8 @@
 // ---------------------------------------------------------------------
 
 // If use litElement base class
-import { LitElement, html, css } from "lit-element";
-import { classMap } from "lit-html/directives/class-map";
+import { LitElement, html, css } from "lit";
+import { classMap } from "lit/directives/class-map.js";
 
 // Import touch detection lib
 import "focus-visible/dist/focus-visible.min.js";
```
Comment (blackfalcon) [inline src/auro-flightline.js:11]: Remove this line

Comment (fajar-apri-alaska) [reply src/auro-flightline.js:11]: The focus-visible one?

**Comment by fajar-apri-alaska** [comment]
I have pushed some more changes, please re-review @blackfalcon.
Thanks~

**Review by blackfalcon — CHANGES_REQUESTED** [review]
Please see task for missing update

- https://github.com/AlaskaAirlines/auro-flightline/issues/53

File: package.json [hunk 8]
```diff
@@ -17,113 +17,107 @@
...
+    "build:api": "wca analyze 'src/auro-*.js' --outFiles docs/api.md",
     "bundler": "rollup -c",
     "bundle:test": "rollup -c -w",
-    "postinstall": "node packageScripts/postinstall.js",
+    "postinstall": "node packageScripts/postinstall.mjs",
     "cssLint": "stylelint \"./src/*.css\"",
-    "dev": "npm run sassBuild:watch",
-    "distJS": "copyfiles -u 1 -V './src/**/*.js' ./dist",
-    "sassRender": "sass-render src/*.css -t ./scripts/staticStyles-template.js",
+    "dev": "concurrently --kill-others 'npm run build:watch' 'npm run serve'",
```
Comment (blackfalcon) [inline package.json:102]: Forgot to add the `build:watch` npm script 

https://github.com/AlaskaAirlines/WC-Generator/blob/master/template/package.json#L138

**Review by blackfalcon — CHANGES_REQUESTED** [review]
Please also address this missing update

- https://github.com/AlaskaAirlines/auro-flightline/issues/54

**Review by Izook — APPROVED** [review]
Yugi Thumb

**Review by blackfalcon — APPROVED** [review]

**Pull request merged**